src/pipeline.ml

```diff
@@ -129,25 +142,26 @@ module Make (OCurrent : S.OCURRENT) = struct
             opam @@
             begin match os_family with
             | `Cygwin | `Linux ->
-              copy ~chown:"opam:opam" ~src:["."] ~dst:"/home/opam/opam-repository" () @@
+              copy ~link:true ~chown:"opam:opam" ~src:["."] ~dst:"/home/opam/opam-repository" () @@
               run "opam-sandbox-disable" @@
               run "opam init -k local -a /home/opam/opam-repository --bare" @@
-              run "rm -rf .opam/repo/default/.git"
+              run "rm -rf .opam/repo/default/.git" @@
+              copy ~link:true ~src:["Dockerfile"] ~dst:"/Dockerfile.opam" ()
```

Comment:
It's just to be sure to understand: why do you need this file whereas before it wasn't required? And, why the Dockerfile becomes a `.opam` file?